scripts/consume_sitemaps.py

```diff
@@ -16,8 +18,11 @@
 )
 from isb_web import sqlmodel_database
 from isb_web.sqlmodel_database import SQLModelDAO
+from isb_lib.models.thing import Thing
+from itertools import islice
+import asyncio
 
-CONCURRENT_DOWNLOADS = 20
+CONCURRENT_DOWNLOADS = 100
```

Comment:
Perf was about the same 20 at a time vs. 100 at a time, which leads me to believe that the bottleneck isn't in the network calls, but rather the db work.